### DISCUSSION TITLE
Ranks Placeholders

### DISCUSSION THREAD

**Discussion by gazdyy** [Zrips/CMI#6898] [Enhancement]
Please add placeholder for counting people in each rank

**Comment by CloudeLecaw** [comment]
You mean the amount of people that have the rank on server?

**Comment by gazdyy** [comment]
I mean placeholder like... %cmi_rankcount_newbie% %cmi_rankcount_explorer% amount of people in separately each rank

**Comment by CloudeLecaw** [comment]
So basicly you mean `%cmi_rank_count_<rank>%`

**Comment by gazdyy** [comment]
yes
